scout/commands/delete/delete_command.py

```diff
@@ -8,7 +8,7 @@
 LOG = logging.getLogger(__name__)
 
 BYTES_IN_ONE_GIGABYTE = 1073741824  # (1024*1024*1024)
-DELETE_VARIANTS_HEADER = "Case n.\tNcases\tInstitute\tCase name\tCase ID\tCase track\tAnalysis date\tTotal variants\tRemoved variants"
+DELETE_VARIANTS_HEADER = "Case n.\tNcases\tInstitute\tCase name\tCase ID\tCase track\tAnalysis date\tStatus\tResearch\tTotal variants\tRemoved variants"
```

Comment:
See, thats why the join is so good.. 😄

Reply:
Right I'll fix also the header like that!